src/components/dropdown/index.js

```diff
@@ -20,11 +20,11 @@ export default class Dropdown extends Component {
         let {props, state} = this
 
         const items = {
-            text: (a,b) => <li key={a} onClick={this.onSelect.bind(this, b)}>{props.options[b]}</li>,
-            color: (a,b) => <div key={a} onClick={this.onSelect.bind(this, b)} style={{backgroundColor: props.options[b]}}></div>
+            text: (a,b) => <li key={a} onClick={this.onSelect.bind(this, b)} className={style.item}>{props.options[b]}</li>,
+            color: (a,b) => <li key={a} onClick={this.onSelect.bind(this, b)} className={style.color} style={{backgroundColor: props.options[b]}}></li>
```

Comment:
now everything is the same but `style` and `className` you can fold the rest, (DRY)